keyboards/clickety_split/leeloo/keymaps/default/rules.mk

```diff
@@ -1 +1,2 @@
-LTO_ENABLE = yes
```

Comment:
This file should be removed.

Reply:
Thank you @waffle87, I'll remove the file.